JS/Lesson34/goksel/recipe-app/src/modules/user/LogIn/index.tsx

```diff
@@ -0,0 +1,130 @@
+import Avatar from '@mui/material/Avatar';
+import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
+import { Box, Button, Checkbox, Container, FormControl, FormControlLabel, FormHelperText, Grid, IconButton, InputAdornment, InputLabel, OutlinedInput, TextField, Typography } from '@mui/material';
+import Visibility from '@mui/icons-material/Visibility';
+import VisibilityOff from '@mui/icons-material/VisibilityOff';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { useForm } from "react-hook-form";
+
+interface LogInProps {
+    isSignedIn: boolean;
+    setIsSignedIn: any;
+}
+
+export const LogIn = ({ isSignedIn, setIsSignedIn }: LogInProps) => {
+
+    interface DataProps {
```

Comment:
Move this interface outside the component

Reply:
Fixed